src/Umbraco.Core/Persistence/Repositories/ITrackedReferencesRepository.cs

```diff
@@ -0,0 +1,14 @@
+using System;
+using System.Collections.Generic;
+using Umbraco.Cms.Core.Models;
+using Umbraco.Cms.Core.Models.Entities;
+
+namespace Umbraco.Cms.Core.Persistence.Repositories
+{
+    public interface ITrackedReferencesRepository
+    {
+        IEnumerable<RelationItem> GetPagedRelationsForItems(int[] ids, long pageIndex, int pageSize, bool filterMustBeIsDependency,out long totalRecords);
```

Comment:
Would be great to have xdoc comments on public interfaces

Reply:
Addressed in https://github.com/umbraco/Umbraco-CMS/commit/1c7f1b17a0e76bc1fa29926ecaf3abffbe166a7e

Reply:
Great xdocs!